observations/obs_converters/SIF/SIF_to_obs_netcdf.rst

```diff
@@ -0,0 +1,204 @@
+
+
+Standard workflow:
+
+#. Download the Level 3 data for the months of interest. Years 2002-2018 are available
+   as of 5/18/21.  (see 'Data Source' below)
+#. Make note of the SIF wavelength the data is centered upon. This information is 
+   included in the SIF variable of netcdf file ``SIF_740_daily_corr``  
+#. Build the DART executables with support for land observations. This is done by running 
+   ``preprocess`` with ``obs_def_lnd_mod.f90`` in the list of ``input_files`` for 
+   ``preprocess_nml``.
+#. Provide basic information via the ``SIF_to_obs_netcdf_nml`` (e.g. verbose, wavelength)
+#. Convert single or multiple SIF netcdf data files using ``SIF_to_obs_netcdf``
+#. Combine all output files for the region and timeframe of interest into one file using
+   :doc:`../../../assimilation_code/programs/obs_sequence_tool/obs_sequence_tool`
+
+For some models (CLM, for example), it is required to reorganize the observation sequence 
+files into a series of files that contains ONLY the observations for each assimilation. 
+This can be achieved with the `makedaily.sh <makedaily.sh>`__ script.
+
+Namelist
+--------
+
+This namelist is read from the file ``input.nml``. Namelists start with an ampersand '&' 
+and terminate with a slash '/'.  Character strings that contain a '/' must be enclosed in
+quotes to prevent them from prematurely terminating the namelist.
+
+::
+
+   &SIF_to_obs_netcdf_nml
+      input_file_list = 'SIF.input.txt',
+      verbose         = 0
+      wavelength      = 740
+      /
+
+
+.. container::
+
+   +-----------------+--------------------+-----------------------------------------------------------------------------+
+   | Contents        | Type               | Description                                                                 |
+   +=================+====================+=============================================================================+
+   | input_file_list | character(len=256) | Name of the Level 3 netcdf containing with SIF data. This may be a          |
+   |                 |                    | relative or absolute filename.                                              |
+   +-----------------+--------------------+-----------------------------------------------------------------------------+
+   | verbose         | integer            | Print more/less information during the ``SIF_to_obs_netcdf`` execution.     |
+   +-----------------+--------------------+-----------------------------------------------------------------------------+
+   | wavelength      | integer            | The wavelength at which SIF irradiance is centered upon (e.g. 740 nm)       | 
+   +-----------------+--------------------+-----------------------------------------------------------------------------+
+
+
+Data sources
+------------
+
+The data was acquired from the following `site <https://cornell.app.box.com/s/gkp4moy4grvqsus1q5oz7u5lc30i7o41/folder/100438579357>`__,
```

Comment:
```suggestion
The datasets are available from the `Cornell University Box service <https://cornell.app.box.com/    s/gkp4moy4grvqsus1q5oz7u5lc30i7o41/folder/100438579357>`__,
```
"site" as the link reads a bit weird, so suggest having the name of the service as the link.